content/fr/shift-left.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Shift Left
+status: Completed
+category: Concept
+tags: ["methodologie", "", ""]
+---
+
+Le terme "Left" dans "Shift Left" fait référence aux premiéres étapes dans le cycle de vie du développement logiciel,
+en envisageant ce cycle comme une ligne où les étapes sont exécutées de gauche à droite.
+Le "Shift Left" est la pratique consistant à mettre en œuvre des tests, la sécurité ou d'autres pratiques de développement
```

Comment:
Et donc pareil ici et pour la suite.
Le "Shift Left", L'approche "Shift Left", La méthodologie "Shift Left"  ???

Reply:
fix in https://github.com/cncf/glossary/pull/2956/commits/95a67215a8afdd86e0941c28204a28466bc805b6